packages/docs/stories/GoalsAndGuidelines.mdx

```diff
@@ -19,27 +19,32 @@ We build on trusted vendors, proven standards and the expertise of our community
 These principles help us to achieve our goals.
 
 
-## Mission and goals
+## Vision, mission and goals
```

Comment:
Wording seems to clash with the headline before ("Vision and principles of Synergy")

Reply:
Changed somehow ;)